File: src/main/java/cn/xihan/http/request/HeadRequest.java
package cn.xihan.http.request;


import cn.xihan.http.model.HttpMethod;

/**
 *    author : Android 轮子哥
 *    github : <a href="https://github.com/getActivity/EasyHttp">https://github.com/getActivity/EasyHttp</a>
 *    time   : 2020/10/07
 *    desc   : Head 请求
 */
public final class HeadRequest extends UrlRequest<HeadRequest> {

    public HeadRequest() {

    }


    @Override
    public String getRequestMethod() {
        return HttpMethod.HEAD.toString();
    }
}

File: src/main/java/cn/xihan/lib/easyhttp/request/HeadRequest.kt
package cn.xihan.lib.easyhttp.request

import cn.xihan.lib.easyhttp.model.HttpMethod

/**
 * author : Android 轮子哥
 * github : [https://github.com/getActivity/EasyHttp](https://github.com/getActivity/EasyHttp)
 * time   : 2020/10/07
 * desc   : Head 请求
 */
class HeadRequest : UrlRequest<HeadRequest?>() {
    override val requestMethod: String
        get() = HttpMethod.HEAD.toString()
}